### PR TITLE
submariner bug

### PR DESCRIPTION
**Pull request by jeswanke**
https://github.com/stolostron/console/pull/1712

Signed-off-by: John Swanke <jswanke@redhat.com>

https://github.com/stolostron/backlog/issues/22622

### CODE REVIEW

File: frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx [hunk 1]
```diff
@@ -378,23 +378,12 @@ export function InstallSubmarinerForm(props: { availableClusters: Cluster[] }) {
...
-                },
-                spec: {
-                    installNamespace: 'submariner-operator',
-                },
-            })
             const broker: Broker = {
                 apiVersion: BrokerApiVersion,
                 kind: BrokerKind,
                 metadata: {
                     name: 'submariner-broker',
```
Comment (KevinFCormier) [inline frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx:385]: Please use the constant `defaultBrokerName`.

**Review by KevinFCormier — CHANGES_REQUESTED** [review]
See inline comment. 

Also, I think there might be more issues with the original PR.

What happens if `isGlobalnetAlreadyConfigured` is `true` but we are adding unsupported clusters now? Seems like we will add a Broker resource incorrectly.

File: frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx (hunk 1 above)
Comment (jeswanke) [reply frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx]: thanks @dberg1 !

**Pull request merged**